js/index.js

```diff
@@ -33,4 +33,50 @@ for (let i=0; i < skills.length; i++){
 
 skill.innerText = skills[i];
 skillsList.appendChild(skill);
-}
\ No newline at end of file
+}
+//message forms
+const messageForm = document.forms.leave_message;
+
+messageForm.addEventListener('submit', function(event) {
+
+    event.preventDefault();
+
+    const usersName = event.target.usersName.value;
+    const usersEmail = event.target.usersEmail.value;
+    const usersMessage = event.target.usersMessage.value;
+
+    console.log('Name:', usersName);
+    console.log('Email:', usersEmail);
+    console.log('Message:', usersMessage);
+
+    ////
+
+ const messageSection = document.getElementById('messages');
+ const messageList = messageSection.querySelector('ul');
+ const newMessage = document.createElement('li');
+
+ newMessage.innerHTML = `
+     <a href="mailto:${usersEmail}">${usersName}</a>: 
+     <span>${usersMessage}</span>`;
+
+ const removeButton = document.createElement('button');
+
+ removeButton.innerText = 'remove';
+ removeButton.type = 'button';
+
+
+ removeButton.addEventListener('click', function() {
+
+     const entry = removeButton.parentNode;
+     entry.remove();
+ });
+
+
+ newMessage.appendChild(removeButton);
+ messageList.appendChild(newMessage);
+
+    ////
```

Comment:
What is this for?

Reply:
Just to see the message on the page and be able to remove it. I will fix it for the final project